stable/coredns/templates/configmap-autoscaler.yaml

```diff
@@ -15,9 +15,13 @@ metadata:
     kubernetes.io/name: "CoreDNS"
     {{- end }}
     app.kubernetes.io/name: {{ template "coredns.name" . }}-autoscaler
-{{- if .Values.customLabels }}
-{{ toYaml .Values.customLabels | indent 4 }}
-{{- end }}
+    {{- if .Values.customLabels }}
+    {{- toYaml .Values.customLabels | nindent 4 }}
```

Comment:
ah neat, didn't know about nindent. :+1:

Reply:
It's really useful with `toYaml` which we use everywhere :D